README.md

```diff
@@ -0,0 +1,77 @@
+# Using cra-template-truemark
+
+## How can we install cra-template-truemark? - Step by Step Guide
+
+1. Clone using the following repo link
+
+   ```md
+   git clone https://github.com/TruemarkDev/cra-template-truemark.git for https.
+
+   git clone git@github.com:TruemarkDev/cra-template-truemark.git for ssh.
+   ```
+
```

Comment:
I think we have publish this template to npm. That way, we can simple install the template.

Reply:
since there is no template.json file , installing the template wont be working anymore.

Reply:
May I know we decided to do it?
As far as I understand it should be working as any other template.

Reply:
I was asked to remove template.json and include all the packages from template.json to packages.json and now we need to use it by cloning the repo. I had talked about it with Prabin dai during the sprint planning as well and he said that we will be using this repo by cloning it and later change the url.